server/athenian/api/openapi/openapi.yaml

```diff
@@ -5079,3 +5155,105 @@ components:
           - $ref: '#/components/schemas/DeveloperSet'
           - description: Release publishers.
       type: object
+    JIRAMetricID:
+      description: |-
+        Currently supported JIRA activity metrics.
+        * `jira-bug-open` - number of bug-like issues active by the end of the time interval.
```

Comment:
active meaning non-resolved yes.

Also, we should be able to get the timeline of the open bugs during the time period selected. Not sure if that would come with the daily granularity or not?

Reply:
Yes, granularity exists especially for that.